quickstarts/dojo/config.yml

```diff
@@ -0,0 +1,46 @@
+slug: dojo
+
+description: |
+  ## Comprehensive monitoring quickstart for Dojo
+  Monitor your Dojo application performance in the most complex and distributed environments to deliver exceptional user experience by proactively identifying and resolving performance issues without any downtime.
```

Comment:
This sentence is a bit of word salad. I'd recommend just replacing this with the intro from the related doc:

"With the help of New Relic's Dojo dashboard, you can track your application's core web vitals, how much time your users spend on your site, and critical browser performance metrics like initial page load and pages with Javascript errors. "